src/components/container/BloodPressureVisualization/BloodPressureVisualization.stories.tsx

```diff
@@ -0,0 +1,69 @@
+import React from "react"
+import { ComponentStory, ComponentMeta } from "@storybook/react"
+import BloodPressureVisualization, { BloodPressureVisualizationProps } from "./BloodPressureVisualization"
+import Card from "../../presentational/Card"
+import Layout from "../../presentational/Layout"
+import queryBloodPressure, { BloodPressureDataParameters } from "../../../helpers/blood-pressure-data-providers/query-blood-pressure"
+import { previewBloodPressureDataPoint } from './BloodPressureVisualization.previewdata'
+
+export default {
+	title: "Container/BloodPressureVisualization",
+	component: BloodPressureVisualization,
+	parameters: {
+		layout: 'fullscreen',
+	}
+} as ComponentMeta<typeof BloodPressureVisualization>;
+
+const Template: ComponentStory<typeof BloodPressureVisualization> = (args: BloodPressureVisualizationProps) =>
+	<Layout>
```

Comment:
I would add colorScheme="auto" to this story and make sure it works in dark mode (you can change your computer to dark mode fairly easily)...in particular will ensure we use the various CSS variables for text / border colors since those are set up for light/dark mode